app/main/posts/views/main.html

```diff
@@ -16,8 +16,6 @@ <h1 class="mode-context-title"><a href="/" ng-bind="nav.site.name"></a></h1>
             <div class="mode-context-body">
                 <p markdown-to-html="nav.site.description"></p>
 
```

Comment:
this was legacy stuff that survived the initial merges but should have gone away much earlier. Basically replaced by the active search filters directive